src/node_i18n.cc

```diff
@@ -461,6 +461,9 @@ int32_t ToUnicode(MaybeStackBuffer<char>* buf,
                                   &status);
   }
 
+  // https://url.spec.whatwg.org/#concept-domain-to-unicode
+  info.errors &= ~UIDNA_ERROR_EMPTY_LABEL;
```

Comment:
Hmm this is a bit tricky. A more detailed comment like the comment should be warranted:

```js
// UTS #46's ToUnicode operation applies no validation of domain name length
// (nor an flag requesting it to do so, like VerifyDnsLength for ToASCII). For
// that reason, unlike ToASCII below, ICU4C correctly accepts long domain
// names. However, ICU4C still sets the EMPTY_LABEL error in contrary to UTS
// #46. Therefore, explicitly filters out that error here.
```